exporter/dynatraceexporter/metrics_exporter.go

```diff
@@ -259,7 +265,8 @@ func (e *exporter) sendBatch(ctx context.Context, lines []string) error {
 func (e *exporter) start(_ context.Context, host component.Host) (err error) {
 	client, err := e.cfg.HTTPClientSettings.ToClient(host.GetExtensions(), e.settings)
 	if err != nil {
-		return err
+		e.settings.Logger.Sugar().Error("failed to construct HTTP client", zap.Error(err))
```

Comment:
Keep log casing the same for each line :) 

Since logs are intended to be read by humans, they should start with a capital case :)